modules/core/src/shaderlib/shader-module.ts

```diff
@@ -0,0 +1,53 @@
+// luma.gl
+// SPDX-License-Identifier: MIT
+// Copyright (c) vis.gl contributors
+
+import {Sampler, Texture} from '@luma.gl/core';
+
+export type BindingValue = Buffer | Texture | Sampler;
```

Comment:
// Maybe add a comment that this is intended to be upstreamed and replace the same file in luma.gl

Reply:
In the PR description I noted that this is a PoC and that I plan on making these changes in luma